sci-mathematics/cgal/cgal-4.11.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+CMAKE_BUILD_TYPE=Release
```

Comment:
This is actually only necessary because you dropped the existing `cgal-4.7-fix-buildsystem.patch`. If it did not apply anymore, please adapt it to the changes in 4.11.1 to be able to drop the override here again.

Reply:
About this one I am not sure. Upstream has now the following in there:
```
if ( NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug"
     AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "MinSizeRel" )
  message( FATAL_ERROR "${CMAKE_BUILD_TYPE} is not a valid build type: only Release, Debug, RelWithDebInfo or MinSizeRel is allowed" )
endif()
```
So I think this should cover it. It rather seems that Gentoo sets this to `Gentoo` somewhere?

Reply:
Well, it was just appended with two new conditions. `cmake-utils.eclass` makes some decisions based on the build type, you can look it up if you want to.

Reply:
what i mean is that too me it looks like this upstream thing covers our specific patch already. isnt that the case?

Reply:
Sorry, looked at the patch again ;)